.travis.yml

```diff
@@ -1,8 +1,26 @@
 language: python
+python:
+  - "2.7"
+  - "3.5"
 
-matrix:
-  include:
-    - env: LINT_CHECK
-      python: "2.7"
-      install: pip install flake8
-      script: flake8
+install:
+  - sudo apt-get update
+  - if [[ "$TRAVIS_PYTHON_VERSION" == "2.7" ]]; then
```

Comment:
This is not required, since you are creating an env with `$TRAVIS_PYTHON_VERSION` anyways.

Reply:
To reiterate, this is fine, but redundant since you are creating a conda environment with the version pinned to $TRAVIS_PYTHON_VERSION. So you are better off downloading only Miniconda3.

Reply:
Done